src/prepublicationUtils.js

```diff
@@ -36,14 +36,16 @@ export function fieldRefersToKoneellisestiTuotettuTietue(field) {
 
 
 export function fieldRefersToTarkistettuEnnakkotieto(field) {
-  return containsSubstringInSubfieldA(field, 'TARKISTETTU ENNAKKOTIETO');
+  return containsSubstringInSubfieldA(field, 'TARKISTETTU ENNAKKOTIETO') || containsSubstringInSubfieldA(field, 'arkistettu ennakkotieto');
```

Comment:
Can this function `containsSubstringInSubfieldA` be refactored so that there is optional parametter to ignore case? Like if true use regexp flag i?

Reply:
Done.